pom.xml

```diff
@@ -184,6 +186,93 @@ under the License.
           </delimiters>
           <useDefaultDelimiters>false</useDefaultDelimiters>
         </configuration>
+        <executions>
+          <execution>
+            <id>copy-anchor-js-into-build-output</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <resources>
+                <resource>
+                  <directory>${project.build.directory}/node_modules/anchor-js</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+              <outputDirectory>${minifiyResourcesDirectory}/js</outputDirectory>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-async-js-into-build-output</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <resources>
+                <resource>
+                  <directory>${project.build.directory}/node_modules/jquery/dist</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+              <outputDirectory>${minifiyResourcesDirectory}/js</outputDirectory>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-bootstrap-into-build-output</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <resources>
+                <resource>
+                  <directory>${project.build.directory}/node_modules/bootstrap/dist</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+              <outputDirectory>${minifiyResourcesDirectory}</outputDirectory>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-resources-into-build-output</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <resources>
+                <resource>
+                  <directory>${basedir}/src/main/resources</directory>
+                  <filtering>false</filtering>
+                </resource>
+              </resources>
+              <outputDirectory>${minifiyResourcesDirectory}</outputDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <!-- Lets use the frontend plugin to grab the release of bootstrap. -->
+      <plugin>
+        <groupId>com.github.eirslett</groupId>
+        <artifactId>frontend-maven-plugin</artifactId>
```

Comment:
does this work on windows?

Reply:
Its been a while since I've had a windows license for development, many years ago I had it working on a Windows Jenkins Build Agent. You can see Node.JS supports all platforms here: https://nodejs.org/dist/v16.15.1/ I believe the plug-in uses your OS/Arch to work out which one to retrieve.